portal/static/js/i18next-config.js

```diff
@@ -21,110 +21,114 @@ var __i18next = window.__i18next = (function() {
                 window.localStorage.removeItem("i18nextLng");
             }
         }
-        options = options || {};
-        if (options.lng) {
-            options.lng = options.lng.replace("_", "-");
-        }
-        /*
-         * consuming translation json from each corresponding locale
-         */
-        var source = options.loadPath ? options.loadPath : "/static/files/locales/{{lng}}/translation.json";
-
-        i18next.use(i18nextXHRBackend) /* global i18next i18nextXHRBackend */
-            .init({
-                fallbackLng: options.fallbackLng ? options.fallbackLng.replace("_", "-") : "en-US",
-                lng: options.lng ? options.lng : "en-US",
-                preload: options.lng ? [options.lng] : false,
-                debug: options.debug ? options.debug : (getQueryString["debugi18next"] ? true : false),
-                ns: options.ns ? options.ns : ["translation"],
-                defaultNS: "translation",
-                initImmediate: options.initImmediate ? options.initImmediate : false,
-                keySeparator: "----",
-                nsSeparator: "----",
-                load: "currentOnly", //this reads language code in en-US, en-AU format
-                returnEmptyString: false,
-                returnNull: false,
-                saveMissing: true,
-                missingKeyHandler: function(lng, ns, key, fallbackValue) {
-                    if (options.missingKeyHandler) {
-                        options.missingKeyHandler(lng, ns, key, fallbackValue);
+        var source = options.loadPath ? options.loadPath : "/static/files/locales/{{lng}}/translation.json"; //consuming translation json from each corresponding locale
```

Comment:
Are you able to use `||` here? A little easier to read than a ternary, but not a big deal :)
eg
```
var source = options.loadPath || "/static/files/locales/{{lng}}/translation.json";
```